src/create-node-middleware.ts

```diff
@@ -1,16 +1,17 @@
-import { RequestListener } from "http";
-import { createNodeMiddleware as createWebbhooksMiddleware } from "@octokit/webhooks";
+import type { RequestListener } from "http";
+import { createNodeMiddleware as createWebhooksMiddleware } from "@octokit/webhooks";
 
-import { ApplicationFunction } from "./types";
-import { MiddlewareOptions } from "./types";
+import type { ApplicationFunction } from "./types";
+import type { MiddlewareOptions } from "./types";
```

Comment:
Lets merge these 2 imports together
```suggestion
import type { ApplicationFunction, MiddlewareOptions } from "./types";
```

Reply:
done